lib/levels.js

```diff
@@ -119,7 +120,23 @@ function getLevel (level) {
 function isLevelEnabled (logLevel) {
   const { values } = this.levels
   const logLevelVal = values[logLevel]
-  return logLevelVal !== undefined && (logLevelVal >= this[levelValSym])
+  return logLevelVal !== undefined && this[levelCompSym](logLevelVal, this[levelValSym])
+}
+
+function compareLevel (direction, current, expected) {
+  if (direction === 'DESC') {
+    return current <= expected
+  }
+
+  return current >= expected
+}
+
+function genLevelComparison (levelComparison) {
```

Comment:
Please add jsdoc blocks to these functions.

Reply:
Done